doc/source/cluster/config.rst

```diff
@@ -63,15 +63,25 @@ The ``example-full.yaml`` configuration is enough to get started with Ray, but f
         InstanceType: p2.8xlarge
 
 **Docker**: Specify docker image. This executes all commands on all nodes in the docker container,
-and opens all the necessary ports to support the Ray cluster. It will also automatically install
-Docker if Docker is not installed. This currently does not have GPU support.
+and opens all the necessary ports to support the Ray cluster. This currently does not have GPU support.
```

Comment:
does it really still not have GPU support?

Reply:
I'm going to test that. I think it might

Reply:
Actually, I think GPUs *do* work.